packages/protocol/test-sol/unit/common/FeeHandler.t.sol

```diff
@@ -19,6 +19,8 @@ import "@celo-contracts/uniswap/test/MockUniswapV2Factory.sol";
 import "@celo-contracts/uniswap/test/MockERC20.sol";
 import "@mento-core/test/mocks/MockSortedOracles.sol";
 import "@mento-core/test/mocks/MockReserve.sol";
+import "@celo-contracts/common/ProxyFactory.sol";
+import "@celo-contracts/governance/GovernanceApproverMultiSig.sol";
```

Comment:
I understand the ProxyFactory import is so that foundry compiles it, if that's the case I'll add a comment.
What is the GovernanceApproverMultiSig import for?

Reply:
Same reason - without it it was throwing unknown artifact